mac/hid.c

```diff
@@ -682,6 +683,7 @@ hid_device * HID_API_EXPORT hid_open_path(const char *path)
 {
 	hid_device *dev = NULL;
 	io_registry_entry_t entry = MACH_PORT_NULL;
+	IOReturn ret;
```

Comment:
```suggestion
	IOReturn ret = kIOReturnSuccess;
```

Reply:
Committed initialization, but with kIOReturnInvalid to prevent occasional future misuse.

Reply:
yes, that's fine too